gradio/components.py

```diff
@@ -480,6 +480,7 @@ class Number(Changeable, Submittable, IOComponent):
     Creates a numeric field for user to enter numbers as input or display numeric output.
     Preprocessing: passes field value as a {float} or {int} into the function, depending on `precision`.
     Postprocessing: expects an {int} or {float} returned from the function and sets field value to it.
+    Examples: Float or int representing value.
```

Comment:
For consistency, let's bracket the type. We'll probably want to italicize the types or render them a little differently on the docs page
```suggestion
    Examples: {float} or {int} representing the value to populate the component.
```

Reply:
(Same for all of the other components ofc)

Reply:
Sounds good! I will file an issue to apply different formatting for the types so we don't forget.